### ISSUE TITLE
Assertion violation at Simplex.cc:409

### ISSUE THREAD

**Issue by rainoftime** [usi-verification-and-security/opensmt#110]
Hi, for the following formula,
~~~~
(set-logic QF_LIA)
(declare-fun _substvar_297_ () Bool)
(declare-fun _substvar_302_ () Int)
(declare-fun _substvar_305_ () Bool)
(declare-fun _substvar_312_ () Bool)
(declare-fun v1 () Bool)
(declare-fun i0 () Int)
(declare-fun v6 () Bool)
(declare-fun i5 () Int)
(declare-fun v11 () Bool)
(declare-fun v17 () Bool)
(declare-fun v19 () Bool)
(declare-fun v24 () Bool)
(assert (or v19 (= (* (/ 692 4) i5) 692)))
(assert (or v24 v17 _substvar_297_))
(assert (or (= v6 v1) (distinct (= i5 (* i0 (/ 692 4))) v11)))
(assert _substvar_312_)
(assert (> _substvar_302_ 4))
(assert _substvar_305_)
(assert (=> v17 v11))
(assert (<= 793 (* i0 (/ 692 4))))
(check-sat)
~~~~

opensmt commit 134e367
~~~~
 opensmt: /home/opensmt/src/tsolvers/lasolver/Simplex.cc:409: bool Simplex::invariantHolds() const: Assertion `false' failed.
 Non-basic (column) LRA var v0 has value (793/173 | 0) <= (4/173 | 1/173)
~~~~

**Comment by rainoftime** [comment]
~~~~
(set-logic QF_LIA)
(declare-fun _substvar_136_ () Bool)
(declare-fun v2 () Bool)
(declare-fun i3 () Int)
(declare-fun i4 () Int)
(declare-fun v15 () Bool)
(declare-fun i5 () Int)
(declare-fun v29 () Bool)
(declare-fun v43 () Bool)
(assert _substvar_136_)
(assert (> i5 920))
(assert (< 528 (* (/ i4 61) 59)))
(assert (or v43 v2))
(assert (or (xor v15 v29) (> i5 920) (distinct i4 (- i3))))
(assert (distinct i4 (- i3)))
(check-sat)
~~~~

**Comment by blishko** [comment]
Fixed by commit 46814dd9 in #111
